docs/general/attributes.md

```diff
@@ -306,12 +306,13 @@ Examples of `peer.service` that users may specify:
 
 ## General identity attributes
 
-These attributes may be used for any operation with an authenticated and/or authorized enduser.
+These attributes may be used for any operation with an authenticated or anonymous enduser.
 
 <!-- semconv identity -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `enduser.id` | string | Username or client_id extracted from the access token or [Authorization](https://tools.ietf.org/html/rfc7235#section-4.2) header in the inbound request from outside the system. | `username` | Recommended |
+| `enduser.anon_id` | string | Randomly generated id of an anonymous user that is persisted throughout the lifetime of the visits. | `V1StGXR8_Z5jdHi6B-myT` | Recommended |
```

Comment:
I'd like to see it called out (a bit more) that this value can be any (attribute) "valid" value such as a GUID.

Here is a description of this field (which we called a different name as part of our planning), which has the same meaning as what you are introducing. here.

Client RUM description: Anonymous user id - will be the same for a given browser session. It will persist across page navigations in the same browser session

Value: String / GUID

Reply:
I wonder if we should also clarify that this is different than a session. The user ID will will persist across page navigations like [session ID](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/session.md#attributes). However, session can expire while the user ID will not. In other words, user ID will be the same across multiple sessions. Am I understanding this correctly?

Reply:
This is true, but I think semantic conventions only what the attribute represents and not any other details.